AIDungeon_2.ipynb

```diff
@@ -36,49 +38,200 @@
         "\n",
         "## How to play\n",
         "1. Click \"Tools\"-> \"Settings...\" -> \"Theme\" -> \"Dark\" (optional but recommended)\n",
-        "2. Click \"Runtime\" -> \"Run all\"\n",
-        "3. Wait until all files are downloaded (only has to be one once, and it will take some time)\n",
-        "4. It will then take a couple minutes to boot up as the model is loaded onto the GPU. \n",
-        "5. If you have questions about getting it to work then please [go to github repo](https://github.com/AIDungeon/AIDungeon) to get help. \n",
+        "2. Go to **Main Game** section below\n",
+        "3. Run Install block\n",
+        "3. Run Download Model block \n",
+        "4. It will then take a couple minutes to boot up as the model is downloaded loaded onto the GPU. \n",
+        "5. Run the game block \n",
+        "6. If you have questions about getting it to work then please [go to github repo](https://github.com/AIDungeon/AIDungeon) to get help. \n",
         "\n",
         "## About\n",
         "* While you wait you can [read adventures others have had](https://aidungeon.io/)\n",
-        "* [Read more](https://pcc.cs.byu.edu/2019/11/21/ai-dungeon-2-creating-infinitely-generated-text-adventures-with-deep-learning-language-models/) about how AI Dungeon 2 is made.",
-        "- **[Support AI Dungeon 2](https://www.patreon.com/bePatron?u=19115449) on Patreon to help me to continue improving the game with all the awesome ideas I have for its future!**\n"
+        "* [Read more](https://pcc.cs.byu.edu/2019/11/21/ai-dungeon-2-creating-infinitely-generated-text-adventures-with-deep-learning-language-models/) about how AI Dungeon 2 is made.- **[Support AI Dungeon 2](https://www.patreon.com/bePatron?u=19115449) on Patreon to help me to continue improving the game with all the awesome ideas I have for its future!**"
```

Comment:
Can we break this back into 2 lines?

Reply:
Will do. I don't know what happened there, forgot to double check.